### PR TITLE
Separate preparing label function and typos

### PR DESCRIPTION
**Pull request by toir427**
Splitting the same lines of code used in two places into a single separate function and some typos

### CODE REVIEW

File: framework/widgets/ActiveField.php [hunk 1]
```diff
@@ -829,7 +815,6 @@ protected function getClientOptions()
         if ($clientValidation) {
             $validators = [];
             foreach ($this->model->getActiveValidators($attribute) as $validator) {
-                /* @var $validator \yii\validators\Validator */
```
Comment (WinterSilence) [inline framework/widgets/ActiveField.php:832]: instead it move up

File: framework/widgets/ActiveField.php [hunk 2]
```diff
@@ -962,4 +947,21 @@ protected function addErrorClassIfNeeded(&$options)
             Html::addCssClass($options, $this->form->errorCssClass);
         }
     }
+
+    /**
+     * @param array $options
+     * @return array
+     */
+    public function preparingLabel(&$options)
```
Comment (WinterSilence) [inline framework/widgets/ActiveField.php:955]: `protected function preparingLabel(array &$options)`

File: framework/widgets/ActiveField.php [hunk 3]
```diff
@@ -962,4 +948,21 @@ protected function addErrorClassIfNeeded(&$options)
             Html::addCssClass($options, $this->form->errorCssClass);
         }
     }
+
+    /**
+     * @param array $options
+     * @return array
+     */
+    protected function preparingLabel(&$options)
```
Comment (machour) [inline framework/widgets/ActiveField.php:956]: prepareLabel() sounds better IMHO

Comment (machour) [inline framework/widgets/ActiveField.php:956]:
```suggestion
    protected function prepareLabel(&$options)
```

File: framework/widgets/ActiveField.php [hunk 4]
```diff
@@ -603,19 +596,12 @@ public function checkbox($options = [], $enclosedByLabel = true)
...
         } else {
-            if (isset($options['label']) && !isset($this->parts['{label}'])) {
-                $this->parts['{label}'] = $options['label'];
-                if (!empty($options['labelOptions'])) {
-                    $this->labelOptions = $options['labelOptions'];
-                }
-            }
-            unset($options['labelOptions']);
-            $options['label'] = null;
+            $options = $this->preparingLabel($options);
```
Comment (machour) [inline framework/widgets/ActiveField.php:604]:
```suggestion
            $options = $this->prepareLabel($options);
```

File: framework/widgets/ActiveField.php [hunk 5]
```diff
@@ -551,19 +551,12 @@ public function radio($options = [], $enclosedByLabel = true)
...
```
Comment (machour) [inline framework/widgets/ActiveField.php:559]:
```suggestion
            $options = $this->prepareLabel($options);
```

**Comment by toir427** [comment]
Thanks, @machour and @WinterSilence

File: framework/widgets/ActiveField.php (hunk 1 above)
Comment (toir427) [reply framework/widgets/ActiveField.php:832]: Why should I use it? That can get a Validator.
image

Comment (WinterSilence) [reply framework/widgets/ActiveField.php:832]: @toir427 1. not all use phpstorm 2. code inspections work better with annotations

Comment (toir427) [reply framework/widgets/ActiveField.php:832]: ok

**Pull request closed without merging**